src/ctia/store.clj

```diff
@@ -2,13 +2,13 @@
 
 (defprotocol IActorStore
   (read-actor [this id])
-  (create-actor [this new-actor])
+  (create-actor [this new-actors])
```

Comment:
Discussed in channel.  The bulk codepath in ES is used in Face and Kiries with single events, alot.  No problems observed.